content/en/news/releases/1.20.x/announcing-1.20/_index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Announcing Istio 1.20.0
+linktitle: 1.20.0
+subtitle: Major Release
+description: Istio 1.20 Release Announcement.
+publishdate: 2023-11-14
+release: 1.20.0
+aliases:
+- /news/announcing-1.20
+- /news/announcing-1.20.0
+---
+
+We are pleased to announce the release of Istio 1.20. This is the last Istio release of 2023. We would like to thank the
+entire Istio community for helping get the 1.20.0 release published. We would like to thank the Release Managers for
+this release, `Xiaopeng Han` from DaoCloud, `Aryan Gupta` from Google, and `Jianpeng He` from Tetrate. The release
+managers would specially like to thank the Test & Release WG lead Eric Van Norman (IBM) for his help and guidance
+throughout the release cycle. We would also like to thank the maintainers of the Istio work groups and the broader Istio
+community for helping us throughout the release process with timely feedback, reviews, community testing and for all
+your support to help ensure a timely release.
+
+{{< relnote >}}
+
+{{< tip >}}
+Istio 1.20.0 is officially supported on Kubernetes versions `1.25` to `1.28`.
+{{< /tip >}}
+
+## What's new
+
+### Full Support for Gateway API GA (v1.0.0) and Improvements
+
+The Kubernetes [Gateway API](http://gateway-api.org/) is an initiative to bring a rich set of service networking APIs (
+similar to those of Istio VirtualService and Gateway) to Kubernetes.
+
+Now Kubernetes Gateway API is GA, and Istio brings full support for it! This effort was a widespread community effort
+across the broader Kubernetes ecosystem and has multiple conformant implementations (including Istio). This marks a
+significant milestone, as Istio users can now leverage the stable set of Gateway API
+features for enhanced traffic management and ingress control in production environments.
+
+Check out the [mesh documentation](/docs/tasks/traffic-management/ingress/gateway-api/#mesh-traffic) to get started.
+
+In this release, we have also added support for configuring Istio
+CRDs `AuthorizationPolicy`, `RequestAuthentication`, `Telemetry` and `WasmPlugin` for Kubernetes Gateway API via
+the `targetRef` field.
+
+### Ambient Mesh TODO(hanxiaop：ambient content and status)
```

Comment:
@howardjohn @linsun could you please check the `What's new` section and give me some suggestions on what needs to be added or removed? Also, I'm not sure about the ambient content. Thanks!

Reply:
@howardjohn @linsun Is there anything related to ambient that needs to be added in 1.20?